autometa/taxonomy/get_lca_taxa.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
```

Comment:
This needs a header like the other scripts in the repo. Also a short description of what the script should be doing should be here as well.